app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,23 @@
+class FriendshipsController < ApplicationController
+	
+	def index
+		@user = User.find(params[:id]) || current_user
+		@user.friends
```

Comment:
Hi guys. What did we need this line for?

Also please check #8

Reply:
Well, we want to have a separate page for friend list. I was going to do it with a friends method in user's controller, but this is better, I guess?